test/e2e/performanceprofile/functests/utils/node_inspector/inspector.go

```diff
@@ -31,8 +31,6 @@ const clusterRoleSuffix = "cr"
 const clusterRoleBindingSuffix = "crb"
 
 var initialized bool
-var namespace *corev1.Namespace = namespaces.NodeInspectorNamespace
-var nodeInspectorName = testutils.NodeInspectorName
```

Comment:
we use this idiom when we want the name to be overridable, instead of always using the hardcoded default. It seems to me in this case we don't need this option, so the change should be fine